sdk/trace/provider.go

```diff
@@ -350,15 +351,115 @@ func WithSampler(s Sampler) TracerProviderOption {
 // are used used by the Tracers the TracerProvider and the Spans they create
 // to limit tracing resources used.
 //
-// If this option is not used, the TracerProvider will use the default
-// SpanLimits.
+// If this or WithRawSpanLimits are not provided, the TracerProvider will use
+// the value set for the corresponding environment variable:
+//
+// • AttributeValueLengthLimit: OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT
+//
+// • AttributeCountLimit: OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT
+//
+// • EventCountLimit: OTEL_SPAN_EVENT_COUNT_LIMIT
+//
+// • AttributePerEventCountLimit: OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT
+//
+// • LinkCountLimit: OTEL_SPAN_LINK_COUNT_LIMIT
+//
+// • AttributePerLinkCountLimit: OTEL_LINK_ATTRIBUTE_COUNT_LIMIT
+//
+// If both options are not provided and no environment variable is set, the
+// TracerProvider will use the default limit value:
+//
+// • AttributeValueLengthLimit: unlimited
+//
+// • AttributeCountLimit: 128
+//
+// • EventCountLimit: 128
+//
+// • AttributePerEventCountLimit: 128
+//
+// • LinkCountLimit: 128
+//
+// • AttributePerLinkCountLimit: 128
+//
+// If any filed of sl is zero or negative it will be replaced with the default
+// value for that field.
+//
```

Comment:
I think Mauve might be the best color 😉 

```suggestion
// If this or WithRawSpanLimits are not provided, the TracerProvider will use
// the value set for the corresponding environment variable or its default value:
//
// • AttributeValueLengthLimit: OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT
//   Default: unlimited
//
// • AttributeCountLimit: OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT
//   Default: 128
//
// • EventCountLimit: OTEL_SPAN_EVENT_COUNT_LIMIT
//   Default: 128
//
// • AttributePerEventCountLimit: OTEL_EVENT_ATTRIBUTE_COUNT_LIMIT
//   Default: 128
//
// • LinkCountLimit: OTEL_SPAN_LINK_COUNT_LIMIT
//   Default: 128
//
// • AttributePerLinkCountLimit: OTEL_LINK_ATTRIBUTE_COUNT_LIMIT
//   Default: 128
//
// If any filed of SpanLimits is zero or negative it will be replaced with the
// default value for that field.
//
```